docs/network-policy.md

```diff
@@ -236,8 +241,83 @@ ephemeral and unpredictable.
 - Rules assume the priority in which they are written. i.e. rule set at top
   takes precedence over a rule set below it.
 
+## Antrea NetworkPolicy
+
+Antrea NetworkPolicy is another Policy CRD, which is similar to the
+ClusterNetworkPolicy CRD, however its scope is limited to a Namespace.
+The purpose of introducing this CRD is to allow admins to take advantage of
+advanced NetworkPolicy features and apply them within a Namespace to
+complement the K8s NetworkPolicies. Similar to ClusterNetworkPolicy resource,
+Antrea NetworkPolicy can also be associated with Tiers.
+
+**Note**: Antrea NetworkPolicy is currently in "Alpha" stage and is enabled
+along with Tiers and ClusterNetworkPolicy as part of the `AntreaPolicy`
+feature gate.
+
+### The Antrea NetworkPolicy resource
+
+An example Antrea NetworkPolicy might look like this:
+```
```

Comment:
add `yaml` tag for code block (same for existing CNP example)

Reply:
done